build.gradle.kts

```diff
@@ -45,9 +45,9 @@ extensions.configure<ExtraPropertiesExtension>("ext") {
 gradlePlugin {
     plugins.create("conventionsPlugin") {
         id = "io.github.gradle.gradle-enterprise-conventions-plugin"
-        implementationClass = "com.gradle.enterprise.conventions.GradleEnterpriseConventionsPlugin"
-        displayName = "Gradle Enterprise Conventions Plugin"
-        description = "Gradle Enterprise Conventions Plugin"
+        implementationClass = "com.gradle.enterprise.conventions.DevelocityConventionsPlugin"
```

Comment:
Should we move the relevant classes to the `io.github.gradle` package and remove the `enterprise` part while we are at it?

We should also consider [JEP 261](https://openjdk.org/jeps/261) and make an effort to avoid the package name collision risk with develocity itself

Reply:
I don't see how a separate PR is better than separate commits in this PR?